doc/docker_quick_start.rst

```diff
@@ -0,0 +1,63 @@
+Docker Quick Start
+==========================
+
+How to add a new asset to Ralph and discover information about the added device in Docker Container
+
+
+Add new asset
+----------------
+Open the Assets tabs with you can manage provides advanced Asset management system.
+
+Then select "Add device"
+
+.. image:: _static/docker_quick_start_add_asset.png
+
+Fill the form with sample data
+
+.. image:: _static/docker_quick_start_add_asset_form.png
+    :width: 600px
+
+**Basic-info:**
+
+- Model: blade-server
+- Warehouse: Warehouse #1
+- Service catalog: Default service
+- Environment: Default Environment
+- Region: Default Region
+- Hostname: localhost
+- IP address: 127.0.0.1
+
+**Additional Info:**
+
+- Data center: DC1
+- Server room: Server #1
+- Rack: Rack2
+- Position: 1
+- Slot number: 1
+- Ralph device id: ae3816a0
+- SN/SNs: ae3816a0-ce08-11e4-b832-0242ac110090
+- Barcode/Barcodes: ae3816a0-ce08-11e4-b832-0242ac110090
+
+
+Next Quick Scan IP Address
+----------------
+
+Ralph is able to periodically scan your all networks/environments and detect hardware information, software configuration and location.
```

Comment:
But in this case we will use One-shot quickscan feature to scan localhost as an example.